docs/infrastructure/deployment-targets/reflected-targets/index.md

```diff
@@ -0,0 +1,88 @@
+---
+title: Reflected Targets
+description: Octopus resources created in cloud providers can be discovered automatically by Octopus
+position: 100
+---
+
+Octopus can automatically discover deployment targets during deployment using tags added to cloud resources. These targets that are automatically discovered will also be removed from Octopus when the associated cloud resource is removed. Automatic discovery will take place before the relevant deployment step and can be used as part of deployment processes that create your cloud infrastructure before deployment.
+
+:::hint
+Using automatic discovery of deployment targets is recommended over the existing [dynamic infrastructure](/docs/infrastructure/deployment-targets/dynamic-infrastructure) functionality.
+:::
+
+To enable automatic discovery and clean up of targets use the following steps:
```

Comment:
Suggestion: I'd love for these documents to read like our goal:

> Just configure your credentials, and ensure your cloud resources are tagged

That would obviously still need the detail, but I'd like for that to be the first things users read. 

Then after that we could add a "For existing projects" section and talk about the need to enable dynamic infrastructure and set deployments without targets settings if not already done.

This would help new users not having to chew over our legacy to get to what they wanted.

Reply:
Yeah nice I like it 👍